src/hera.cpp

```diff
@@ -325,7 +320,8 @@ evmc_result hera_execute(
 
       if (msg->kind == EVMC_CREATE && !result.isRevert && hasWasmPreamble(result.returnValue)) {
```

Comment:
I'd just add the `hera->metering` check here to simplify the code (and avoid a copy).

Reply:
Perhaps its good to keep the `assert > 5`, so in that case must use `move` on `returnValue`.